### PR TITLE
Timetable draft

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: slides/components/Timetable/Timetable.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class Timetable extends React.Component {
```
Comment (okonet) [inline slides/components/Timetable/Timetable.js:3]: Not sure why all that is needed. Here is the timetable that's is faster to update and it renders the same thing: https://github.com/component-driven/component-driven-development/blame/master/slides/cdds-worskhop.mdx#L24-L30

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -116,3 +118,53 @@ function Code({ children, language, inline = false }) {
...
+        {
+            title: "Maintain! Pattern Journey",
+            length: "2:00"
+        },
+        {
+            title: "Recap",
+            length: "0:30"
+        }
+    ]}
+/>
```
Comment (okonet) [inline slides/ds101-worskhop.mdx:170]: Let's not focus on not important things for now. Timetable isn't moving us closer to the completion.

File: slides/components/Timetable/Timetable.js (hunk 1 above)
Comment (varya) [reply slides/components/Timetable/Timetable.js:3]: I don' get. Your link is to markdown, it does not calculate times.

Comment (okonet) [reply slides/components/Timetable/Timetable.js:3]: And why do we need that?

Comment (okonet) [reply slides/components/Timetable/Timetable.js:3]: I don't remember discussing anything like this nor I don't see any value of spending time on that so I'm wondering what's the purpose of that

File: slides/ds101-worskhop.mdx (hunk 2 above)
Comment (varya) [reply slides/ds101-worskhop.mdx:170]: 1. It's already done. We loose no time merging it.
2. It is helpful to understand how long every part of the workshop takes and how long the whole thing gets. In my previous experience, it has always been a hassle. Usually people tend to plan a lot while timing is limited. We have to shape the exercises keeping this in mind. Summing up times is a nice reminder.

Comment (okonet) [reply slides/ds101-worskhop.mdx:170]: We went this route with the ReasonConf website and we ended up spending 2 man-days fixing issues caused by users looking at the time table from different time zones. Maybe we won't get those issues but I'm really reluctant to merge something that has non-zero maintenance const and low value.

> Usually people tend to plan a lot while timing is limited. We have to shape the exercises keeping this in mind. Summing up times is a nice reminder.

I don't believe that having this data in JSON going to help us and it's easy enough to count the duration so I would rather not a technical solution to a simple problem.

**Pull request closed without merging**